spec/yabeda/tags_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.describe Yabeda::Tags do
+  describe '.build' do
+    subject { described_class.build(tags) }
+
+    let(:tags) { { controller: 'foo' } }
+
+    context 'when general tags are not set' do
+      before do
+        Yabeda.general_tags.clear     
+      end
+      it { is_expected.to eq({ controller: 'foo' }) }
+    end
+
+    context 'when general tags are set' do
+      before do
+        Yabeda.configure do
+          general_tag :environment, 'test'
+        end
+      end
+
```

Comment:
Let's add spec for merge behavior (I think it is useful):


```suggestion

context "when current tags are the same as general tags" do
  let(:tags) { { environment: 'production' } }

  it { is_expected.to eq({ environment: 'production' }) }
end

```

Reply:
Agree - it's great case to add spec

Reply:
Done